src/containers/PharmacyList/PharmacyList.tsx

```diff
@@ -123,16 +128,56 @@ export function PharmacyList(props: Props) {
         addressSegments.push(address.province);
         addressSegments.push(postalCodeToHumanFormat(address.postcode));
       }
-      const isBooking = pharmacy.vaccineAvailabilities.length > 0;
+
+      const vaccineAvailabilitiesWithAvailable: VaccineAvailabilityTimeslotRequirementExpandedResponse[] = pharmacy.vaccineAvailabilities.filter(
+        (vaccineAvailability) => {
+          return vaccineAvailability.numberAvailable > 0;
+        },
+      );
+
+      interface RequirementsWithAvailabilityInterface
+        extends VaccineAvailabilityRequirementsResponse {
+        numberAvailable: number;
+      }
+
+      interface VaccineAvailabilitiesByDateAndRequirementsInterface {
```

Comment:
This is duplicated from `PharmacyCard` - it is common for the parent component to need to know the shape of the props for the child component it is rendering. There are two approaches to tackle this:

Better: Export the interface from the child, and import it into the parent

Best: Have the parent infer the props of the child e.g. `type PharmacyCardProps = React.ComponentProps<typeof PharmacyCard>;`